packages/blockchain-wallet-v4-frontend/src/services/CheckAccountZeroBalance/index.ts

```diff
@@ -0,0 +1,19 @@
+import { SwapAccountType, SwapCoinType } from '../../data/components/swap/types'
+
+const checkAccountZeroBalance = (
+  accounts: { [key in SwapCoinType]: Array<SwapAccountType> }
+): boolean => {
+  let allAccounts: Array<SwapAccountType> = []
+
+  for (const coin in accounts) {
+    for (const account of accounts[coin]) {
+      allAccounts = [...allAccounts, account]
+    }
+  }
+
+  return allAccounts.every(
+    account => account.balance === 0 || account.balance === '0'
```

Comment:
could just do `account.balance == 0`?

Reply:
Because it can be "0" from the server. I could make it as 

```
Number(account.balance) === 0
```

But then I would assume that this would evaluate as a number, and it can not be always the case.